kythe/cxx/common/indexing/KytheGraphRecorder.cc

```diff
@@ -94,7 +94,8 @@ static const std::string* kEdgeKindSpellings[] = {
     new std::string("/kythe/edge/ref/call/direct"),
     new std::string("/kythe/edge/ref/call/direct/implicit"),
     new std::string("/kythe/edge/denotes"),
-    new std::string("/kythe/edge/named")};
+    new std::string("/kythe/edge/named"),
+    new std::string("/kythe/edge/typed/init")};
```

Comment:
Would it be helpful to add experimental to these names to indicate they may change or go away?